manifest.json

```diff
@@ -8,20 +8,17 @@
       "default_popup": "popup.html"
   },
   "background": {
-        "service_worker": "blockpage.js"
+        "service_worker": "blockpage.js",
+        "type": "module"
   },
   "icons": {
       "16": "assets/icon16.png",
       "48": "assets/icon48.png",
       "128": "assets/icon128.png"
   },
   "permissions": [
```

Comment:
Please provide some form of reasoning for removing other permissions

Reply:
We do not require those for the extension yet

Reply:
Ok will merge once others have been resolved